src/components/molecules/ContentSearchbar.vue

```diff
@@ -0,0 +1,140 @@
+<template>
+	<div class="search">
+		<div :class="[isActive ? 'input-active' : '', 'search__container']">
+			<input
+				ref="searchInput"
+				v-autowidth="{ maxWidth: '960px', minWidth: '0px', comfortZone: 40 }"
```

Comment:
```suggestion
				v-autowidth="{
					minWidth: 'var(--size-content-width-min)',
					maxWidth: 'var(--size-content-width-max)',
					comfortZone: 40,
				}"
```

40 seems to be a magic number. Can you explain me why you have chosen `40`? ([Why magic numbers are bad](https://stackoverflow.com/questions/47882/what-is-a-magic-number-and-why-is-it-bad))